commands/preupgrade/__init__.py

```diff
@@ -23,6 +23,11 @@
              help='Set preferred channel for the IPU target.',
              choices=['ga', 'tuv', 'e4s', 'eus', 'aus'],
              value_type=str.lower)  # This allows the choices to be case insensitive
+@command_opt('flavour', default=upgrade_path.LEAPP_UPGRADE_FLAVOUR_DEFAULT,
+             choices=[upgrade_path.LEAPP_UPGRADE_FLAVOUR_DEFAULT, upgrade_path.LEAPP_UPGRADE_FLAVOUR_SAP_HANA],
+             help='Specify upgrade path flavor')
+# NOTE(ivasilev) choices argument here will depend on flavor, so not specifying defaults
+@command_opt('to', help='Specify RHEL version to upgrade to')
```

Comment:
We are using `source`/`original` and `target` keywords everywhere - presentations, docs, reports, .... let's stick to it. `to` is not so much descriptive.

```suggestion
@command_opt('target', help='Specify RHEL version to upgrade to')
```

Reply:
Ok, will change.